src/input.cpp

```diff
@@ -499,10 +499,19 @@ void input_manager::add_gamepad_keycode_pair( int ch, const std::string &name )
     gamepad_keyname_to_keycode[name] = ch;
 }
 
-void input_manager::add_mouse_keycode_pair( const int ch, const std::string &name )
+void input_manager::add_mouse_keycode_pair( const MouseInput mouse_input, const std::string &name )
 {
-    mouse_keycode_to_keyname[ch] = name;
-    mouse_keyname_to_keycode[name] = ch;
+    mouse_keycode_to_keyname[static_cast<int>( mouse_input )] = name;
+    mouse_keyname_to_keycode[name] = static_cast<int>( mouse_input );
+}
+
+void input_manager::add_mouse_keycode_pair_translated
+( const MouseInput mouse_input, const std::string &key_name_untranslated )
+{
+    add_mouse_keycode_pair(
+        mouse_input,
+        translate_marker_context( "key name", key_name_untranslated )
```

Comment:
Good idea, but I'm pretty sure you can't actually do that. `translate_marker_context` must appear with the actual text extracted, otherwise it won't extract it.

Reply:
Thank for valuable feedback! My bad,  I haven't checked what translate_marker_context() does.

Removed ```add_mouse_keycode_pair_translated()``` entirely.
Now Im using the same trick as ```keyboard_code_keycode_pair``` and ```add_keyboard_code_keycode_pair``` do